### ISSUE TITLE
Restrict to left-Alt for window dragging

### ISSUE THREAD

**Issue by ttencate** [ttencate/taekwindow#20]
Converted from [SourceForge issue 1653698](http://sourceforge.net/support/tracker.php?aid=1653698), submitted by slagdogg

Have been using taekwindow for several days now. Overall I love it, but noticed that in Photoshop the Alt + click actions are pretty primary. For example, removing part of a selection with the marquee tool.

I think a nice compromise would be if takewindow only used left Alt for window dragging, then if a user finds themselves in a tool which requires Alt dragging they can still use the right Alt key.

**Comment by ttencate** [comment]
Submitted by slagdogg

Logged In: YES 
user_id=1679966
Originator: NO

Sounds like an excellent idea. I'd been having trouble with Photoshop as well :) As soon as I've finished the configuration tool I'll look into this.

**Comment by ttencate** [comment]
Submitted by slagdogg

Logged In: YES 
user_id=610849
Originator: NO

Hi, you might want to test this version I patched:
http://tfpsly.free.fr/Files/taekwindow-0.1.1PatchedMaya.zip

I quickly added a fix for Maya and Photoshop, looking for the name of the application. For these applications, I disabled Taekwindow. Not in a cleaned way, as the names of the windows should not be stored in the source code, but it does the job.

See http://sourceforge.net/tracker/index.php?func=detail&aid=1738690&group_id=185457&atid=913356
